cookbooks/swift/templates/default/etc/swift/object-server/default.conf-template.erb

```diff
@@ -3,6 +3,8 @@ mount_check = false
 disable_fallocate = true
 workers = 1
 log_facility = LOG_LOCAL2
+# NOTE: servers_per_port necessary for replicator/reconstructor so must go here
+servers_per_port = <%= @servers_per_port %>
```

Comment:
Line 70 (on the right of the diff?) at
https://review.openstack.org/#/c/184189/13/swift/obj/replicator.py

On Wed, Jun 10, 2015 at 12:38 PM, clayg <notifications@github.com> wrote:

> In
> cookbooks/swift/templates/default/etc/swift/object-server/default.conf-template.erb
> <https://github.com/swiftstack/vagrant-swift-all-in-one/pull/16#discussion_r32157374>
> :
>
> > @@ -3,6 +3,8 @@ mount_check = false
> >  disable_fallocate = true
> >  workers = 1
> >  log_facility = LOG_LOCAL2
> > +# NOTE: servers_per_port necessary for replicator/reconstructor so must go here
> > +servers_per_port = <%= @servers_per_port %>
>
> I don't yet understand why replicator/reconstructor need this - I get why
> they might need bind_ip...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swiftstack/vagrant-swift-all-in-one/pull/16/files#r32157374>
> .
>